controllers/users.js

```diff
@@ -500,6 +500,28 @@ const nonVerifiedDiscordUsers = async (req, res) => {
   return res.json(data);
 };
 
+const updateRoles = async (req, res) => {
+  try {
+    const result = await userQuery.fetchUser({ userId: req.params.id });
+    if (result?.userExists) {
+      const dataToUpdate = req.body;
+      const userQueryResponse = await userQuery.updateRoles(result.user, dataToUpdate);
+      if (userQueryResponse.isRoleUpdated) {
+        return res.status(200).json({
+          message: "role updated successfully!",
+        });
+      } else {
+        return res.boom.conflict("role already exist!");
+      }
+    } else {
+      return res.boom.notFound("User not found");
+    }
+  } catch (error) {
+    logger.error(`Error while updateRoles: ${error}`);
+    return res.boom.serverUnavailable("Something went wrong please contact admin");
```

Comment:
Can you please use constants for this message because we are using multiple places

Reply:
Done